modules/nw-route-specific-annotations.adoc

```diff
@@ -66,6 +67,15 @@ This value is applicable to re-encrypt and edge routes only. For more informatio
 `if-none`: sets the header if it is not already set.| `ROUTER_SET_FORWARDED_HEADERS`
 
 |===
+[.small]
+--
+1. If the number of IP addresses and CIDR ranges in an allowlist exceeds 64, they are written into a separate file that is then referenced from `haproxy.config`.
```

Comment:
If the number of IP addresses and CIDR ranges in an allowlist exceeds 61, they are written into a separate file that is then referenced from `haproxy.config`. The file will be stored in `/var/lib/haproxy/router/whitelists` folder